cmd/agent/main.go

```diff
@@ -52,6 +56,11 @@ func run(cmd *cobra.Command, args []string) error {
 		snapshotURL = fmt.Sprintf(DefaultSnapshotURLFmt, entrypoint.ExternalSnapshotPort)
 	}
 
+	diagnosticsURL := os.Getenv("AES_DIAGNOSTICS_URL")
```

Comment:
Would it be possible to add an environment variable (or rejigger this one) to completely disable diagnostic reporting? It might be a feature some users want disabled. Think of it as a feature flag?